root/collection/CollectionIndex.js

```diff
@@ -161,8 +158,9 @@ const listPicker = (
 
 const CollectionIndex = (props: Props):
 React.Element<typeof CollectionLayout> => {
+  const $c = React.useContext(CatalystContext);
```

Comment:
I think it's better not to do this for top-level components, since it's always passed as a prop in that case: https://github.com/metabrainz/musicbrainz-server/blob/b036579/root/server/response.js#L100

So it's actually simpler to just use the prop here. I'm not sure if this is the only example in this PR.

Reply:
I rechecked, I think this was the only one. Amended.